app/styles/themes/white-label/_theme.mixins.scss

```diff
@@ -38,22 +34,22 @@
   display: inline-block;
   margin-right: var(--space-2);
   margin-left: calc(var(--space-1) * -1);
-  color: $c-gray;
+  color: rgb(var(--color-gray));
   font-size: $font-size;
   transform: translateY(.08em);
 }
 
 
 // Used for line wrapping text links.
 @mixin inline-accent(
-  $text-color: $c-body-color,
-  $underline-color: $c-body-color,
-  $hover-underline-color: $c-primary,
-  $hover-bg-color: $c-primary,
-  $hover-text-color: $c-white,
+  $text-color: rgb(var(--color-dark-gray)),
+  $underline-color: rgb(var(--color-dark-gray)),
+  $hover-underline-color: rgb(var(--color-reddish-orange)),
```

Comment:
white label doesn't have a `--color-reddish-orange`

Reply:
that color was on this file a lot, that's why I used it - I changed them all to the --color-ember-red instead, but I'm thinking that will need to get changed when we work on white label